lib/services/book.ts

```diff
@@ -0,0 +1,14 @@
+import {prisma} from "@lib/services/db";
+
+export async function getBookWithPosts(isbn: string) {
```

Comment:
I think it could be helpful to have a JSDoc here

Reply:
Done